src/plugins/omake/OMakeEquip.ml

```diff
@@ -331,6 +331,20 @@ let get_lib_deps ?(transitive=false) ?(filter = fun _ -> true) pkg bs =
        bs.bs_build_depends
     )
 
+let get_sorted_trans_lib_deps pkg bs =
```

Comment:
The code doesn't check for duplicates. I think you may want to get rid of the duplicates.